ocs_ci/utility/utils.py

```diff
@@ -3106,6 +3106,35 @@ def skipif_upgraded_from(version_list):
         return False
 
 
+def skipif_no_kms():
+    """
+    This function evaluates the condition to skip a test if the cluster
+    doesn't have configured KMS encryption.
+
+    Return:
+        (bool): True if test needs to be skipped else False
+
+    """
+    from ocs_ci.ocs.resources import storage_cluster
+
+    cluster = storage_cluster.get_storage_cluster()
```

Comment:
Can also be checked against config.DEPLOYMENT.get("kms_deployment") without the further checks.

Reply:
https://github.com/red-hat-storage/ocs-ci/blob/6adc5a4fb5139a801b5a84769fa31056170c83c3/ocs_ci/deployment/deployment.py#L657

Reply:
Right, do you think it should be checked in ocs-ci configuration instead of cluster configuration?
It would save one oc command call but this way it can be ran without specifying additional configuration file in test run to make sure that the test is not skipped if cluster have kms set. It also tests that the cluster has the correct value set in StorageCluster yaml.